contracts/test_contracts/blockinfo_tester/src/blockinfo_tester.cpp

```diff
@@ -0,0 +1,100 @@
+#include <blockinfo_tester/blockinfo_tester.hpp>
+
+#include <cstdio>
+#include <type_traits>
+#include <vector>
+
+#include <eosio/action.hpp>
+#include <eosio/check.hpp>
+#include <eosio/datastream.hpp>
+#include <eosio/name.hpp>
+#include <eosio/print.hpp>
+
+namespace {
+
+namespace block_info = eosiosystem::block_info;
+
+}
+namespace system_contracts::testing::test_contracts::blockinfo_tester {
+
+auto process(get_latest_block_batch_info request) -> latest_block_batch_info_result
+{
+   latest_block_batch_info_result response;
+
+   block_info::latest_block_batch_info_result res =
+      block_info::get_latest_block_batch_info(request.batch_start_height_offset, request.batch_size);
+
+   response.result           = std::move(res.result);
+   response.error_code.value = static_cast<uint32_t>(res.error_code);
+
+   /*
```

Comment:
Do we still need the commented out sections?

Reply:
That's what I had assumed, no problem with leaving it. I just wanted to make sure.